Atomic/util.py

```diff
@@ -26,9 +26,12 @@
 
 def _decompose(compound_name):
     """ '[reg/]repo[:tag]' -> (reg, repo, tag) """
-    reg, repo, tag = '', compound_name, ''
+    reg, repo, tag = '', lstrip(compound_name, '/'), ''
```

Comment:
I am spending too much time in functional programming languages. I certainly do.